nixos/doc/manual/configuration/profiles/minimal.section.md

```diff
@@ -7,3 +7,10 @@ graphical stuff. It's a very short file that enables
 only support the user-selected locale,
 [disables packages' documentation](#opt-documentation.enable),
 and [disables sound](#opt-sound.enable).
+
+With this profile enabled, a lot of package derivations will differ from the
+ones available for download in the binary cache (because they are built with
+X libraries enabled by default), and will thus have to be built locally.
+Hence, while configurations built using this profile may be appropriate for
+machines with low resources, the configurations themselves should still be
+built on more powerful machines.
```

Comment:
It should be mentioned that this is because of noXlibs which the minimal profile enables.